### ISSUE TITLE
Accessibility: Add ALT text for images

### ISSUE THREAD

**Issue by HenrikBengtsson** [RConsortium/rconsortium_website#359]
Please consider adding ALT text to images, e.g. the logos on <https://r-consortium.org/members.html>.

**Comment by codetricity** [comment]
@jcasman 

## implementation

As the alt text is unique to every image, we'll need to go through the images manually.
https://quarto.org/docs/authoring/markdown-basics.html

Example syntax.

```
[![](elephant.png){fig-alt="Alt text"}](https://quarto.org)
```

## testing

https://chatgpt.com/share/6840fb8c-7d28-8013-8fa3-b368b788faae

Many options at link above.  This one seems like an easy way to test:

```
Lighthouse (Built into Chrome)
Open DevTools → “Lighthouse” tab → Run an accessibility audit.

Will flag missing or poor alt text.
```

## timeframe

* we can start with new blog posts immediately and test to make sure the qmd to HTML render is working properly
* I suggest we then work on the https://r-consortium.org/members.html page as this is just a set of logos and would benefit from the alt text the most if they are not already implemented
* after this, we can assess priorities using an accessibility audit tool, but it seems like first Projects, then Events, then News would be a reasonable sequence.

**Comment by codetricity** [comment]
@jcasman I created a new Wiki entry that covers  two aspects of the `alt` text

https://github.com/RConsortium/rconsortium_website/wiki/Tips-for-Writing-and-Publish-Blog-Posts

1. for images on the site or the blog post, the syntax is to use `fig-alt`.  The incorrect syntax is  `![this is wrong](image.png)`.   People may be more familiar using Markdown processors like MkDocs which handles the text inside of `![text goes here]` differently from Quarto.
2.  the `alt` text for the thumbnails of the blog post is handled by `image-alt` inthe yaml.  Example: 
    `image-alt: "banner for unlocking chemical volatility with volcalc"`

**Comment by jcasman** [comment]
@codetricity Confirming we will be using this syntax moving forward. Confirming that text within the square brackets is published as a caption, not alt-text.
